app/components/forum/forum-post-new.js

```diff
@@ -9,20 +9,19 @@ export default class ForumPostNewComponent extends Component {
 
   @action
   async save() {
-    let { content, thread } = this;
```

Comment:
You could also update this line to `= this.args` instead of doing that in `createRecord` below; not sure what's more readable